docs/events.yaml

```diff
@@ -615,8 +615,8 @@ x-events:
     host_name: string
     approved_value: bool
 
-- name: host_cluster_registered
-  message: "Host {host_name}: registered to cluster"
+- name: host_registration_succeeded
+  message: "Host {host_name}: Successfully registered"
```

Comment:
no need for cluster-id in this context ?

Reply:
looked at this again.
I saw no `cluster_id` in `V2RegisterHostParams`, however, I do see `V2RegisterHost()` attempts to fetch the cluster via `getBoundCluster()` so I'll add this as an optional param.

Reply:
Done